pydatastructs/linear_data_structures/tests/benchmarks/test_algorithms.py

```diff
@@ -41,9 +41,6 @@ def test_quick_sort():
 def test_bubble_sort():
     _test_common_sort(bubble_sort, size=2000)
 
-@pytest.mark.xfail
-def test_intro_sort():
-     _test_common_sort(intro_sort, size=2000)
```

Comment:
Why this is removed?

Reply:
There is no CPP backend for this. Shown as an error on my local.

Reply:
Actually skipped on my local due to this.

Reply:
This was added by me in December, back then, I didn't know whether to add it or not, so I had just added.